### ISSUE TITLE
[DE230]: HA Controllers Glance Image Sync Failure

### ISSUE THREAD

**Issue by byronmccollum** [rcbops/chef-cookbooks#496]
#### This is an issue in the `v4.0.0` branch of the cookbooks, which is the default for RPCS. Please do not close saying it was fixed in a different brach.

---

There is a bug in `glance-image-sync.py` (see [#1](https://github.com/rcbops/glance-image-sync/issues/1) and [#3](https://github.com/rcbops/glance-image-sync/issues/3)). There are only a few messages that Glance emits that are of interest to Glance Image Sync. These irrelevant messages are not being filtered out, and cause Glance Image Sync to stop working as soon as it comes across one.

Temporary fix is to override the default:

```
"glance": {
    "replicator": {
        // Force install of newer glance-image-sync
        "checksum": "ba0582359f4bd5d18cc27188b481a13f7e655b55"
    }
}
```

**Comment by mattt416** [comment]
Hi @byronmccollum,

Thanks for creating this issue.  We are currently reviewing the best way of getting such fixes from our grizzly branch into the v4.0.0 branch and will update this ticket with more information once we have something nailed down and implemented.

Thanks,
Matt

**Comment by zfeldstein** [comment]
Glance-image-sync is now included in the glance cookbook:

https://github.com/rcbops-cookbooks/glance/commit/890ee2eca10bd2f4a6a788858c4306d4358da433

Closing this issue.
